app/main.py

```diff
@@ -14,4 +14,4 @@ def count_occurrences(phrase: str, letter: str) -> int:
     :param letter: letter to find occurrences of it
     :return: count occurrences of letter in phrase
     """
-    # write your code here
+    return (phrase.upper().count(letter.upper()))
```

Comment:
There is no need to wrap the final result in parentheses in the return statement, remove them.